src/app/dashboard/artist-corner/form.tsx

```
@@ -0,0 +1,406 @@
+const productSchema = z
+    .object({
+        name: z.string().min(2).max(64),
+        description: z.any(),
+        price: z.union([z.string(), z.number()]),
+        images: z.array(z.string()).min(1).max(5),
+        download: z.string().min(1),
+        is_free: z.boolean()
+    })
+    .superRefine((data, ctx) => {
+        const numericPrice =
+            typeof data.price === 'string' ? parseFloat(data.price) : data.price
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)